tests/e2e/config_secret.go

```diff
@@ -1112,8 +1112,8 @@ var _ = ginkgo.Describe("Config-Secret", func() {
 		10. Cleanup all objects created during the test
 	*/
 
-	ginkgo.It("[csi-config-secret-file] Add wrong targetvSANFileShareDatastoreURLs and switch back to the correct "+
-		"targetvSANFileShareDatastoreURLs", func() {
+	ginkgo.It("Add wrong targetvSANFileShareDatastoreURLs and switch back to the correct "+
+		"targetvSANFileShareDatastoreURLs", ginkgo.Label(P1, vsphereConfigSecret, file), func() {
```

Comment:
Should we label it p0, p1, p2 instead of Capital, as we have followed same norms for other labels ?

Reply:
done